### PR TITLE
Update RESULTS.md for v12

### PR DESCRIPTION
**Pull request by amj**

### CODE REVIEW

File: RESULTS.md [hunk 1]
```diff
@@ -318,5 +318,257 @@ We expect to see it go sideways as it adapts to the new proportion of
 resign-disabled games, then show a similar improvement to the v7a improvement.
 After that, if it flattens off again we'll lower the learning rate further.
 
+After our learning rate cuts, the ratings appeared to taper off.  A brief bit of
+"cyclic learning rate" didn't seem to jog it out of its local minima so we
+called a halt after ~530 models
+
+v7 also marked the beginning of an auto-pair for the evaluation matches that
+would attempt to pair the models about whom our rating algorithm had the
```
Comment (sethtroisi) [inline RESULTS.md]: v7 also marked the beginning of an auto-pair for the evaluation matches that
played more games for models with greater rating uncertainty.

File: RESULTS.md [hunk 2]
```diff
@@ -318,5 +318,257 @@ We expect to see it go sideways as it adapts to the new proportion of
...
+model farthest away in time from among the closest rated.  These made for
+interesting games, as the models of about the same strength from very different
+points in training would take very different approaches.
+
 Our limiting factor continues to be our rate of selfplay.
 
+### v9, TPUS #1, July 19 - August 1
+
+By mid July we were able to get a Cloud TPU implementation working for selfplay and
+TPUs were enabled in GKE, allowing us to attempt running with about 600 TPU
```
Comment (sethtroisi) [inline RESULTS.md]: I remember only having 384 in v9

File: RESULTS.md [hunk 3]
```diff
@@ -318,5 +318,257 @@ We expect to see it go sideways as it adapts to the new proportion of
...
+v2's.  Also, we were able to run our training job on a TPU as well, meaning that
+our training batch size could increase from 256 => 2048, which meant we could
+follow the learning rate schedule published in paper #2.
+
+Unfortunately, TPU code required that we change around where we did the rotation
+of the training examples, moving it from the host to inside the TF graph.
+Unfortunately, I didn't check the default value of the flag and rotating
+training examples was turned OFF until around model 115 or so.
```
Comment (sethtroisi) [inline RESULTS.md]: + After we turned on rotation there was a noticeable bump on the evaluation strength
maybe add photos directory and upload an image

File: RESULTS.md [hunk 4]
```diff
@@ -318,5 +318,257 @@ We expect to see it go sideways as it adapts to the new proportion of
...
+
+Still, this was a hugely successful run:  We were able to run a full 700k steps
+-- the length described in Paper 2 and 3 -- in only two weeks.  14M selfplay
+games were played; more than the 5M in paper2 and less than the 22M in paper 3.
+
+Here too we saw our top models show up after our first rate cut.  But after the
+second rate cut, performance tapered off and the run was stopped.
```
Comment (sethtroisi) [inline RESULTS.md]: -But
After the second rate cut performance variance decreased significantly and didn't seem to improve over the following 150 models

File: RESULTS.md (hunk 3 above)
Comment (amj) [reply RESULTS.md]: i don't have a good solution for hosting photos in a github repo.  it's what the 'pages' or 'wiki' is for, but it's still a little weird... i dunno.  RESULTS isn't really code either :(

**Pull request merged**